backend/hubspot/hubspot.go

```diff
@@ -371,6 +371,27 @@ func (h *HubspotApi) createContactForAdmin(ctx context.Context, email string, us
 }
 
 func (h *HubspotApi) CreateContactCompanyAssociation(ctx context.Context, adminID int, workspaceID int) error {
+	return h.kafkaProducer.Submit(ctx, &kafka_queue.Message{
+		Type: kafka_queue.HubSpotCreateContactCompanyAssociation,
+		HubSpotCreateContactCompanyAssociation: &kafka_queue.HubSpotCreateContactCompanyAssociationArgs{
+			AdminID:     adminID,
+			WorkspaceID: workspaceID,
+		},
+	}, "")
+}
+
+func (h *HubspotApi) CreateContactCompanyAssociationImpl(ctx context.Context, adminID int, workspaceID int) error {
```

Comment:
this makes the association creation async to make sure it happens after the company and contact are created.